SingularityService/src/test/java/com/hubspot/singularity/scheduler/SingularityMachineStatesTest.java

```diff
@@ -333,7 +333,8 @@ public void testFrozenSlaveCanBeDecommissioned() {
 
     saveAndSchedule(request.toBuilder().setSlavePlacement(Optional.of(SlavePlacement.OPTIMISTIC)).setInstances(Optional.of(2)));
```

Comment:
feel free to change this to GREEDY or SEPARATE so that the frozen slave test isn't relying on calculations form optimistic